### ISSUE TITLE
Flatpak application install location should be standardized across Ublue images.

### ISSUE THREAD

**Issue by storyaddict** [ublue-os/main#324]
Discussed this issue with @castrojo, and he suggested I submit it here.

Rebasing between `bazzite` and `bluefin` I now have flatpaks listed under both `user` and `system`. While this may make sense for different applications, if possible, installed flatpaks should be placed in a standard location for all Ublue images.

**Comment by bsherman** [comment]
Universal Blue's  `*-main` and `*-nvidia` images have a goal of staying fairly close to stock Fedora... Thus the comment "with batteries included". We are trying to find the balance of enabling some of the hard (or extremely useful) things which almost every desktop user would want.  See: https://universal-blue.org/scope/#ublue-images-mainnvidia

That said, those images are not mean to be a "new linux distro" but an "easier to use Silverblue/Kinoite" or... a foundation on which to build a more custom version.

As you mention, `bazzite` and `bluefin` are more like a downstream, custom distro. 

Fedora defaults to flatpaks being installed in `system`, and I think `bazzite` may maintain that default behavior.

However, @castrojo specifically was inspired by OpenSuse's MicroOS and how they do per-user flatpak installs. Thus he's opted for `bluefin` to reconfigure (if clicking through the first time installer UI) flatpaks for `user`.


TL;DR: :-) I hear what you are saying, but I believe this is a area of user choice. We likely won't be forcing downstreams to do one or the other.

**Comment by castrojo** [comment]
I'm not wedded to --user in bluefin, but would switch it if it made rebasing easier for people, so I think we should have a discussion about it.

**Comment by akdev1l** [comment]
I don’t think it would affect rebasing either way as flatpaks are stored in /var or /var/home.

**Comment by castrojo** [comment]
Yeah the problem is if you have a bunch of apps installed in bazzite and you rebase to bluefin and then you install stuff there it ends up all mixed together between system and user.

**Comment by castrojo** [comment]
This has been sorted, bazzite and bluefin both do system flatpaks now
